src/plugins/auto/multi_schedule.cpp

```diff
@@ -54,10 +54,11 @@ Pipeline MultiSchedule::GetPipeline(const IInferPtr& syncInferRequest, WorkerInf
             }
         });
     } else {
+        MultiImmediateExecutor::Ptr _FirstExecutor = std::make_shared<MultiImmediateExecutor>();
         pipeline = {
             // if the request is coming with device-specific remote blobs make sure it is scheduled to the specific device only:
             Stage {
-                /*TaskExecutor*/ std::make_shared<IE::ImmediateExecutor>(), /*task*/ [this, &syncInferRequest]() {
+                /*TaskExecutor*/ _FirstExecutor, /*task*/ [this, &syncInferRequest]() {
```

Comment:
let's keep in mind another AR here after guozhong's PR to integrate multi to auto. @WeldonWangwang @wgzintel 
let's add one extra step in this first stage,  disable runtime fallback when remote blob is involved.
like -d AUTO:GPU.0, GPU.1 and the input is remote blob for GPU.0, if fallback to GPU.1, the inference cannot infer anyway, so just disable it and let the exception throw.